app/models/classification.rb

```diff
@@ -36,7 +37,16 @@ def self.scope_for(action, user, opts={})
     when :incomplete
       incomplete_for_user(user).includes(:subjects)
     when :project
-      joins(:project).merge(Project.scope_for(:update, user)).includes(:subjects)
+      return none if opts[:last_id] && !opts[:project_id]
```

Comment:
do we want to raise an error about missing params here instead of returning nothing? E.g. [missing selector params](https://github.com/zooniverse/Panoptes/blob/e4873028032d06c1dab709ef69f34d1157c139a6/lib/subjects/selector.rb#L85)